src/util/map.go

```diff
@@ -10,3 +10,11 @@ func JoinStringMaps(maps ...map[string]string) map[string]string {
 	}
 	return result
 }
+
+// Merge merges two maps. It overwrites existing keys of map1 if conflicting keys exist
+func Merge(map1 map[string]string, map2 map[string]string) map[string]string {
+	for k, v := range map2 {
+		map1[k] = v
+	}
+	return map1
```

Comment:
I don't think we should return map1. maps are like pointers so map1 gets updated automatically.